punchcut/README.md

```diff
@@ -0,0 +1,43 @@
+# punchcut
+
+This is a library for high level loading of glyph outlines (and eventually color outlines and bitmaps)
+from font files. The intention is fully featured (e.g. variations and hinting) support for all glyph sources
+except for the SVG table.
+
+This is part of the [oxidize](https://github.com/googlefonts/oxidize) project.
+
+## Features
+
+Current (✔️), near term (🔜) and planned (⌛) feature matrices:
+
+#### Vector sources:
+
+| Source | Loading | Variations | Hinting |
+|--------|---------|------------|---------|
+| glyf   | ✔️     |  🔜        | ✔️*    |
```

Comment:
IMHO hinting should be planned with a * since it doesn't work at time of writing.